src/components/PageResultsNew/DetailsDrawer.tsx

```diff
@@ -0,0 +1,310 @@
+        },
+        {
+            title: 'CI details',
+            content: (
+                <DrawerPanelBody>
+                    CI team name and contact info (email, IRC, GChat) will be
+                    here.
+                </DrawerPanelBody>
+            ),
+        },
+        {
+            title: (
+                <>
+                    Known issues <Badge isRead>2</Badge>
+                </>
+            ),
+            content: <KnownIssuesTab />,
+        },
+        {
+            title: 'Metadata',
+            content: (
+                <DrawerPanelBody>
+                    Test result metadata will be here (time submitted, UMB
+                    message ID, datagrapper link, ResultsDB link, etc.)
+                </DrawerPanelBody>
+            ),
+        },
+    ];
+
+    // TODO: Persist the drawer width across pages/sessions.
+    const panelContent = (
+        <DrawerPanelContent defaultSize="40rem" isResizable minSize="20rem">
+            <DrawerHead>
+                <Title className="pf-u-pb-sm" headingLevel="h3" size="xl">
+                    {selectedTest?.name}
+                </Title>
+                <DrawerActions>
+                    <DrawerCloseButton onClick={onCloseClick} />
+                </DrawerActions>
+                <Flex
+                    spaceItems={{
+                        default: 'spaceItemsLg',
+                    }}
+                >
+                    {selectedTest?.isWaivable && (
+                        <Button
+                            className="pf-u-p-0"
+                            icon={<ThumbsUpIcon />}
+                            variant="link"
+                        >
+                            Waive
+                        </Button>
+                    )}
+                    <Button
+                        className="pf-u-p-0"
+                        icon={<SyncAltIcon />}
+                        variant="link"
+                    >
+                        Rerun
+                    </Button>
+                    <Button
+                        className="pf-u-p-0"
+                        icon={<BookIcon />}
+                        variant="link"
+                    >
+                        Documentation
+                    </Button>
+                </Flex>
+            </DrawerHead>
+            <DrawerPanelBody className="pf-u-pb-sm">
+                <Alert isInline title="Test information" variant="info">
+                    <TextContent>
+                        <Text>
+                            This test result is provided by the <b>BaseOS QE</b>{' '}
+                            team. If you need help with this test, you can reach
+                            out to the team via <b>#tft on IRC</b>, or via email
+                            at{' '}
+                            <b>
+                                <a href="mailto:baseos-ci@redhat.com">
+                                    baseos-ci@redhat.com
```

Comment:
It is ok to have our emails in code, we are owners, it is our choice.
I think it is better to remove this mail, or we should ask a permission from these team.
I would rather change it to some clear non-existing email.

Reply:
Changed.